### DISCUSSION TITLE
Can't access end entities via RA but can with Public Access Web

### DISCUSSION THREAD

**Discussion by bugz8unny69** [Keyfactor/ejbca-ce#279] [General]
Hi community,

Admittedly, I have no idea what I'm doing. Hence I'm having trouble understanding the discrepancy here. EJBCA is deployed in my home lab to replace XCA. So, I've created a ClientAuth certificate using a new root certificate I created myself. And another ClientAuth certificate using the ManagementCA root certificate. Neither of them can be retrieved from RA UI (aka /ejbca/ra) using a username. But work's fine if I try to retrieve them from the Public Access Web (/ejbca/enrol/keystore.jsp). What am I doing wrong here?

P.S. The error is `End Entity with name 'test' does not exist, status was wrong or enrollment code was incorrect`

Regards,
Horace

**Comment by primetomas** [comment]
I'm not sure if I understand what you are trying to do so.
If I start by assuming that you have the superadmin keystore and you can access the admin UI and RA UI using this...

This video (and others in the channel) may give you some good hints on how to use EJBCA to issue client certificates.
https://www.youtube.com/watch?v=wMD1GgSF-JE

Cheers,
Tomas

**Comment by bugz8unny69** [comment]
I manage to fix it by simply granting an Access Rule to the issuing CA.
